packages/main/src/Tree.ts

```diff
@@ -182,6 +184,21 @@ type WalkCallback = (item: TreeItemBase, level: number, index: number) => void;
 	},
 })
 
+/**
+ * Fired when a tree item is focused.
+ *
+ * @param {HTMLElement} item The focused item.
+ * @public
```

Comment:
Could you double check if this should be a public event.

In the linked feature request I can find
"Please put a @ui5-item-focused=..... binding on the ui5-tree-list in Tree.hbs and fire that same **private** event from Tree.ts"